deploy/crds/crd-certificaterequests.yaml

```diff
@@ -83,7 +83,6 @@ spec:
               description: Desired state of the CertificateRequest resource.
               type: object
               required:
-                - csr
```

Comment:
The `kubebuilder:validation:Required` marker has no effect, which I think is a bug.